pandas/io/pytables.py

```diff
@@ -929,28 +935,30 @@ def func(_start, _stop, _where):
 
     def put(self, key, value, format=None, append=False, **kwargs):
         """
-        Store object in HDFStore
+        Store object in HDFStore.
 
         Parameters
         ----------
         key      : object
         value    : {Series, DataFrame}
-        format   : 'fixed(f)|table(t)', default is 'fixed'
-            fixed(f) : Fixed format
-                       Fast writing/reading. Not-appendable, nor searchable
-            table(t) : Table format
+        format   : 'Fixed(f)|Table(t)', default is 'fixed'
+            Fixed(f) : Fixed format
+                       Fast writing/reading. Not-appendable, nor searchable.
+            Table(t) : Table format
```

Comment:
I think this is incorrect, I know you did it to make the validation of the docstring not fail, but we'll have to find another solution, you can just restore how it was for now.